tests/cli/commands/test_variable_command.py

```diff
@@ -49,6 +49,15 @@ def test_variables_set(self):
         with pytest.raises(KeyError):
             Variable.get("foo1")
 
+    def test_variables_set_with_description(self):
+        """Test variable_set command with optional description argument"""
+        variable_command.variables_set(
+            self.parser.parse_args(["variables", "set", "foo", "bar", "--description", "foo_bar_description"])
+        )
+        assert Variable.get("foo") is not None
```

Comment:
This should also check the description is actually set

Reply:
Yes @uranusjr , you are completely right on this, I was not able to find the getter method of description at that moment. Let me see if I could dig it more to update this test case with the proper validation of the description as well.

Reply:
I am not able to find an existing method to retrieve the variable's `description` in https://github.com/apache/airflow/blob/main/airflow/models/variable.py, should this be considered as a follow up PR with a issue to support this feature ?

Similar to this, I think the CLI command for `variable set` does not contain description as a valid output as well.

Please let me know if you or the team would like to create a dedicated issue to support this feature.

But for this PR, I am inclined to skip this check

Reply:
Thank you @uranusjr , this is helpful I will update my PR to include this in the unit test

Reply:
> Thank you @uranusjr , this is helpful I will update my PR to include this in the unit test

This has been updated in the latest revision, please feel free to review this change.

Thanks